src/lib/package.json

```diff
@@ -73,6 +73,9 @@
         "dest": "../../dist"
     },
     "schematics": "./schematics/collection.json",
+    "dependencies": {
+        "tslib": "^2.0.0"
+    },
     "ng-update": {
         "migrations": "./schematics/migration.json"
     }
```

Comment:
In dieser Datei müssen die Versionen von den Angular packages auch noch hochgezogen werden. Das ist nämlich die package.json die nachher auf npm landet.

Reply:
done